faststream/confluent/security.py

```diff
@@ -14,6 +15,9 @@
 
 
 def parse_security(security: Optional[BaseSecurity]) -> "AnyDict":
+    if security and isinstance(security.ssl_context, ssl.SSLContext):
```

Comment:
I think, we should use `SetupError` instead of basic `ValueError` in wrong broker parameters case

Reply:
Noted. Thanks.